### ISSUE TITLE
centos:7 yum update fails iputils was supposed to be removed but it not

### ISSUE THREAD

**Issue by charlesritchea** [CentOS/sig-cloud-instance-images#72]
```
FROM centos:7
RUN yum update -y && \
    yum upgrade -y && \
    yum install -y boost-regex && \
    yum clean all -y
```
Fails with the following error
iputils-20121221-7.el7.x86_64 was supposed to be removed but is not!

**Comment by fooishbar** [comment]
More detail is available here:
https://jenkins.freedesktop.org/job/docker-images/job/base-images/job/centos-7-x86-64/lastBuild/console

I guess that file needs to be reinstated, or at least an empty decoy in its place ... ?

**Comment by jperrin** [comment]
Can you provide some more information about the backend storage you're using, and the host, etc?

I've run your basics there and I'm unable to duplicate. It works fine in my testing here. I don't see anything about iputils mentioned in the jenkins output either.

**Comment by charlesritchea** [comment]
I'm running KaOS which an Arch distribution, and XFS file system. I think
the FS might be the culprit

**Comment by fooishbar** [comment]
Debian jessie, x86-64, ext4. It doesn't appear to happen every time ... :\ Perhaps an aufs bug.

**Comment by charlesritchea** [comment]
Yes, I've seen that suggested elsewhere. selinux might be a factor as well

**Comment by fooishbar** [comment]
SELinux is not running.

**Comment by jperrin** [comment]
it will 100% be an aufs issue. AUFS is unsupported for these images, as it doesn't support file capabilities, which is used with centos/rhel/fedora. I'd recommend using overlayfs or device-mapper instead of aufs.
